packages/core/src/render3/instructions/defer.ts

```diff
@@ -147,55 +147,71 @@ export function ɵɵdeferPrefetchWhen(rawValue: unknown) {
     const tDetails = getTDeferBlockDetails(tView, tNode);
     if (value === true && tDetails.loadingState === DeferDependenciesLoadingState.NOT_STARTED) {
       // If loading has not been started yet, trigger it now.
-      triggerResourceLoading(tDetails, getPrimaryBlockTNode(tView, tDetails), lView[INJECTOR]!);
+      triggerResourceLoading(tDetails, tView, lView);
     }
   }
 }
 
 /**
- * Sets up handlers that represent `on idle` deferred trigger.
+ * Sets up a logic to handle the `on idle` deferred trigger.
```

Comment:
We should probably be consistent in this language.
```suggestion
 * Sets up logic to handle the `on idle` deferred trigger.
```